ap_src/templates/calendars/calendar.html

```diff
@@ -121,15 +121,15 @@ <h1 class="is-size-5 has-text-weight-bold" id="title-{{ item.team.name }}">{{ it
             <table class="table is-bordered is-striped" style="width: 100%" id="{{ item.team.name }}">
                 <tbody>
                     {% for member in item.team.members %}
+                    {% check_permissions member request.user as editable %}
                     <tr id="{{ member.user.username }}">
                         {% if member.user.username == user.username %}
                         <td class="user-cell">{{user.username}}</td>
                         {% else %}
-                            {% check_permissions member request.user as editable %}
                             {% if editable %}
                             <td>{{member.user.username}}*</td>
                             {% else %}
-                            <td>{{member.user.username}}</td>
+                            <td>{{member.user.username}}*</td>
```

Comment:
Remove the * after `{{member.user.username}}`. That should only appear if the user has permission to alter the person's absences.